akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -920,6 +925,33 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
     }
   }
 
+  def moveJoiningToWeaklyUp(): Unit = {
+    val localGossip = latestGossip
+    val localMembers = localGossip.members
+
+    val enoughMembers: Boolean = isMinNrOfMembersFulfilled
```

Comment:
again, inlining this to the && expression where it used will save unnecessary evaluation (since this val is not used anywhere else)